src/js/components/Data/stories/SpaceX.js

```diff
@@ -181,54 +187,42 @@ export const Table = () => {
       <Box>
         <Data
           properties={{
-            rocket: { label: 'Rocket', options: rocketOptions },
+            rocket: { label: 'Rocket', options: rockets },
             success: { label: 'Success', options: ['Successful', 'Failed'] },
           }}
-          data={docs}
-          total={numberItems}
+          data={result.data}
+          total={total}
+          filteredTotal={result.filteredTotal}
           view={view}
-          onView={(nextView) => {
-            setView(nextView);
-            setPage(1);
-          }}
+          onView={setView}
           toolbar
         >
           <DataTable
             columns={columns}
-            sort={{ ...sort, external: true }}
-            onSort={(opts) => setSort(opts)}
+            sort={{ ...view.sort, external: true }}
+            // TODO: in some other pull request,
+            // integrate sorting between Data and DataTable
+            // onSort={(opts) => setSort(opts)}
           />
+          {result.filteredTotal > view.step && (
```

Comment:
Playing with the filter popup then clearing filters I got it into a state without the pagination controls:
![image](https://user-images.githubusercontent.com/12054362/211923677-b7cc354f-e487-4db6-9659-21b66c4a1fbc.png)

Reply:
I ran into this as well. Steps were:
1. Filter by something
2. Clear filters

Reply:
Good catch. Added `defaultView` to allow the caller to control what "Clear filters" returns to. This allows the caller to persist `step` in the view.